sympy/simplify/simplify.py

```diff
@@ -1166,7 +1166,7 @@ def nsimplify_real(x):
             # We'll be happy with low precision if a simple fraction
             if not (tolerance or full):
                 mpmath.mp.dps = 15
```

Comment:
Is it really a good idea to set this precision to 15. It looks like simple rational numbers with numerators/denominators not greater than 1000 are not really detected correctly. Maybe a lower value like 12 would be more relevant?